src/Admin/BaseMediaAdmin.php

```diff
@@ -155,7 +155,7 @@ public function getObjectMetadata(object $object): MetadataInterface
             $provider->getFormatName($object, MediaProviderInterface::FORMAT_ADMIN)
         );
 
-        return new Metadata($object->getName(), $object->getDescription(), $url);
+        return new Metadata((string) $object, $object->getDescription(), $url);
```

Comment:
What's the purpose of this ?

Reply:
Not 100% sure, but on admin this is already defined. Looks like the media adds a description and the image of the media. 

https://github.com/sonata-project/SonataAdminBundle/blob/master/src/Admin/AbstractAdmin.php#L1703-L1706

Reply:
I just changed it to use $this->toString (like the original admin implementation).